google-cloud-bigtable/lib/google/cloud/bigtable/admin/v2/doc/google/bigtable/admin/v2/bigtable_instance_admin.rb

```diff
@@ -311,6 +311,14 @@ class UpdateAppProfileRequest; end
         #   @return [true, false]
         #     If true, ignore safety checks when deleting the app profile.
         class DeleteAppProfileRequest; end
+
+        # This is a private alpha release of Cloud Bigtable replication. This feature
```

Comment:
Does this "private alpha" language mean that these changes should not be released to the general public?

Reply:
Not sure. I found these comments strange since they are on the public protos, but it doesn't look like we've published this gem yet anyway. There were already comments like this on other classes so I thought it would make sense to get the repos in sync at least.

I don't have any visibility into the release planning, but I suspect we wouldn't want to. Let's check with @theacodes to be sure.

Reply:
The Bigtable team is working on the Ruby client located in this repo. We have given code reviews but don't really have much insight into the design decisions being made.  Perhaps @sduskis can weigh in on whether these updated files are wanted?

Reply:
The comments are out of date.  The "App Profile" feature is now beta.  We need to update those comments in the proto files, and then regenerate GAPIC.